design/src/high-level/creative-automation-case-studies.md

```diff
@@ -0,0 +1,282 @@
+
+### Distinct Paths
+
+Very few options.
+Suffers from many of the same problems as Stardew Valley,
+with lots of manual labor and no way to chain machines together.
+
+### Low Pressure
+
+Extremely low pressure: it's an idle game.
+
+### Compelling Aesthetics
+
+Decent game feel, but very homogenous with no variation on the theme.
+
+## [The Incredible Machine](https://en.wikipedia.org/wiki/The_Incredible_Machine)
+
+**Conclusion:** Great creativity, but no sense of progression and these are simply contraptions rather than automation.
+
+### Interesting Goals
+
+Lots of interesting goals, but secondary goals are basically missing.
+
+Campaign is effectively introductory: map editor mode is the real creative sandbox.
+
+Solving challenges created by other players is often the point.
+
+### Distinct Paths
+
+Incredible for this.
+So many wild gizmos, and possible creative physics-driven interactions.
+
+Lack of secondary goals hurts feeling of distinctness.
+Could be dramatically strengthened with innovations from Opus Magnum.
+
+### Low Pressure
+
+Yep! It's just a puzzle game.
+
+### Compelling Aesthetics
+
+Fantastic: great game feel in the contraptions.
+
+Fun, silly, intuitive components.
+
+## [Opus Magnum](https://store.steampowered.com/app/558990/Opus_Magnum/)
+
+**Conclusion:** Awesome creative automation, but these are mostly just contraptions without a larger purpose.
+
+### Interesting Goals
+
+Pre-baked puzzles, but with really explicit and interesting multi-objective optimization:
+
+- space
+- cycles
+- cost
+- instructions
+
+These are made explicit to the player, and histogram leaderboards encourage competitive optimization.
+
+### Distinct Paths
+
+Fantastic: lots of recipe paths, fantastic constraints and tradeoffs on how to move things around.
+
+### Low Pressure
+
+Great, it's a puzzle game.
+
+### Compelling Aesthetics
+
+Clean and clear aesthetics, cohesive if mostly abstract theme, great whirring and moving contraptions.
+
+## [Oxygen Not Included](https://store.steampowered.com/app/457140/Oxygen_Not_Included/)
+
+**Conclusion:** Good but not great creative automation. Automation options take a long time to unlock. Too many challenges have Best Solutions.
+
+### Interesting Goals
+
+Challenges are largely "satisfy", not "optimize".
+Figuring out how to:
+
+- get food
+- get water
+- get power
+- produce oxygen
+- manage excess heat
+
+is interesting as a new player, but can be dull once the mechanics are understood.
+Very little incentive to actually fine tune working processes.
+
+### Distinct Paths
+
+Lots of good options for each path, mostly restricted by world gen.
+
+Some setups are simpler / more efficient / more reliable (hi Self-Power-Oxygen-Machines) and end up completely dominating over more interesting ones (like Morb farming).
+
+Duplicant behavior can be hard to understand and optimize for in frustrating ways.
+
+Limited tools for genuine automation until quite late in the game.
+
+### Low Pressure
```

Comment:
My understanding is that it's not really possible to make a self-sustaining colony (without the expansion pack), so your colony "will eventually collapse, and you kind of have to accept that" --- how true is that?

If it is, then I'm not ONI does low pressure well?

Reply:
Nah, super easy to make simple self-sustaining colonies. They have heat destruction + geysers and volcanoes for infinite renewable resources.